.github/workflows/pr-validate-label-teamlabel.yml

```diff
@@ -1,12 +1,9 @@
-name: "Validate PRs"
-
+name: "Validate PRs & Assign PR team labels & Pull Request Labeler"
 on:
+  pull_request:
+    branches:
+      - main
   pull_request_target:
```

Comment:
@zomars Would it be safe to club multiple jobs under this event which requires special care in terms of security?